tests/linker/mac/link all/link all-mac.csproj

```diff
@@ -81,6 +81,9 @@
     <Compile Include="..\..\CommonLinkAllTest.cs">
       <Link>CommonLinkAllTest.cs</Link>
     </Compile>
+    <Compile Include="..\..\ios\link all\ILReader.cs">
+      <Link>ILReader.cs</Link>
+    </Compile>

```

Comment:
should be moved to `common` if used for the Mac

Reply:
Done